x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/expression/predicate/logical/BinaryLogic.java

```diff
@@ -34,7 +34,7 @@ protected TypeResolution resolveInputType(Expression e, Expressions.ParamOrdinal
     }
 
     @Override
-    protected Pipe makePipe() {
+    public Pipe makePipe() {
```

Comment:
Why?

Reply:
I'll set this back. I was using it within Wildcard.make_pipe, but since that is unreachable, I'll set this back.